lib/Onyx.js

```diff
@@ -20,6 +21,7 @@ const METHOD = {
 
 // Key/value store of Onyx key and arrays of values to merge
 const mergeQueue = {};
+const mergeQueuePromise = {};
```

Comment:
I followed most of the changes in this PR except for where it came to `mergeQueuePromise`. Could you add some code comments to give a general idea around what it's for and how it's used?

Reply:
This change is mostly for adjusting tests from what I remember. 
Looks like we use it in tests for making sure all the pending merges has been applied.